packages/runtime-common/realm.ts

```diff
@@ -1622,10 +1625,12 @@ export class Realm {
       {
         useWorkInProgressIndex,
         htmlFormat,
+        cardUrls,
       },
     );
 
     let doc = transformResultsToPrerenderedCardsDoc(results);
+    doc.meta.realmInfo = await this.parseRealmInfo();
```

Comment:
Added `realmInfo` to the prerendered cards meta so that we can show realm info (icon, name) where we group prerendered cards by realm in the cards chooser

Reply:
I don't think this should be necessary. We have this information on the client side already and should be available via realm-service.

Reply:
Agree, fixed in https://github.com/cardstack/boxel/pull/1521/commits/d0c565b614f6b1ffa92087e37b80fa9cecdef0c9